src/main/java/com/example/demo/Dao/DaoImpl/StaffDaoImpl.java

```diff
@@ -116,4 +137,71 @@ public List<User> findDepartment(String engineer) {
 		return engineerList; // staffListを呼び出し元に返す（今回はServiceclassに）
 	}
 
+	public User selectOne(String staffCode) {
+
+		Map<String, Object> staff = jdbc.queryForMap("SELECT * FROM staffs WHERE staff_code = ?", staffCode); // sqlテーブルのデータを1件取得;
+																												// //
+																												// sqlテーブルのデータを1件取得
+
+		User user = new User();
```

Comment:
```suggestion
		Map<String, Object> staff = jdbc.queryForMap("SELECT * FROM staffs WHERE staff_code = ?", staffCode); // sqlテーブルのデータを1件取得;

		User user = new User();
```
ソース可読性向上のための改行は1つで事足りるはずですので、不要な改行は削除しておいてください。

Reply:
余分な空行、コメントアウトを削除しました。